src/Raven.Server/ServerWide/ServerStore.cs

```diff
@@ -247,6 +247,33 @@ public void Initialize()
                 }
             };
 
+            try
+            {
+                if (MemoryInformation.IsSwappingOnHddInsteadOfSsd())
+                {
+                    var alert = AlertRaised.Create("Swap Storage Type Warning",
+                        "OS swapping on at least one HDD drive while there is at least one SSD drive on this system. " +
+                        "This can cause a slowdown, consider moving swap-partition/pagefile to SSD",
+                        AlertType.SwappingHddInsteadOfSsd,
+                        NotificationSeverity.Warning,
+                        "Swap Storage Type Warning");
```

Comment:
ohh I see. You mean the key string. But isn't it important to specify a unique key there ? (in order not to get multiple notifications in Studio)